src/plugins/kbn_vislib_vis_types/public/tile_map.js

```diff
@@ -46,54 +48,17 @@ export default function TileMapVisType(Private, getAppState, courier, config) {
 
         pushFilter(filter, false, indexPatternName);
       },
-      mapMoveEnd: function (event) {
-        const agg = _.get(event, 'chart.geohashGridAgg');
-        if (!agg) return;
-
-        agg.params.mapZoom = event.zoom;
-        agg.params.mapCenter = [event.center.lat, event.center.lng];
-
-        const editableVis = agg.vis.getEditableVis();
-        if (!editableVis) return;
-
-        const editableAgg = editableVis.aggs.byId[agg.id];
-        if (editableAgg) {
-          editableAgg.params.mapZoom = event.zoom;
-          editableAgg.params.mapCenter = [event.center.lat, event.center.lng];
-        }
+      mapMoveEnd: function (event, uiState) {
+        uiState.set('mapCenter', event.center);
+        uiState.set('mapZoom', event.zoom);
```

Comment:
I thought it was because during the auto fit it didn't call the zoom function, but after testing it seems I'm mistaken. I'll take it out.